src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -3639,6 +3639,110 @@ public void SetEmptyPropertyToNull()
             }
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]  // desktop doesn't support Package.Open with FileAccess.Write
```

Comment:
You can pass on a string as part of the attribute's constructor, rather than adding a comment 😄 https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.XUnitExtensions/src/Attributes/SkipOnTargetFrameworkAttribute.cs#L17